projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.ts

```diff
@@ -861,6 +854,26 @@ export class NgxLocationPickerComponent implements OnInit, OnChanges, OnDestroy,
     this.setView(coords);
   }
 
+  /**
+   * Adds proximity circle to show the accuracy of received coordinates
+   */
+  private showProximityCircle(coords: number[], accuracy: number): void {
+    this.removeProximityCircle();
+    this.proximityCircle = this.locationViewerService.L.circle(coords, accuracy, { opacity: 0 });
```

Comment:
Wat is die opacity?

Reply:
Die opacity zorgt ervoor dat ervoor dat de rand van de cirkel niet wordt getoond